src/rocstar.js

```diff
@@ -0,0 +1,223 @@
+    const encodedTaskViaProxy = taskViaProxy.method.toHex();
+    const taskViaProxyFees = await taskViaProxy.paymentInfo(turingAddress);
+    const requireWeightAtMost = parseInt(taskViaProxyFees.weight, 10);
+
+    console.log(`Encoded call data: ${encodedTaskViaProxy}`);
+    console.log(`requireWeightAtMost: ${requireWeightAtMost}`);
+
+    console.log(`\nc) Execute the above an XCM from ${shibuyaHelper.config.name} to schedule a task on ${turingHelper.config.name} ...`);
+    const feePerSecond = await turingHelper.getFeePerSecond(SHIBUYA_TOKEN_ID_ON_TURING);
+    const xcmpExtrinsic = shibuyaHelper.createTransactExtrinsic({
+        targetParaId: turingHelper.config.paraId,
+        encodedCall: encodedTaskViaProxy,
+        proxyAccount: proxyAccountId,
+        feePerSecond,
+        instructionWeight: TURING_INSTRUCTION_WEIGHT,
+        requireWeightAtMost,
+    });
+    await sendExtrinsic(shibuyaHelper.api, xcmpExtrinsic, keyPair);
+
+    console.log(`\nAt this point if the XCM succeeds, you should see the below events on both chains:\n
+  1. Shibuya\n
+  xcmpQueue.XcmpMessageSent and polkadotXcm.Sent - an XCM is successfully sent from Shibuya to Turing to schedule a task.\n
+  2. Turing Dev\n
+  a) proxy.ProxyExecuted and automationTime.TaskScheduled - the above XCM is received and executed on Turing.\n
+  b) xcmpHandler.XcmTransactedLocally, xcmpQueue.XcmpMessageSent, xcmpHandler.XcmSent and automationTime.XcmpTaskSucceeded - the task is triggered and its payload is sent to Shibuya via XCM.\n
+  3. Shibuya\n
+  proxy.ProxyExecuted and xcmpQueue.Success - the above payload is received and executed.\n`);
+
+    const taskIdCodec = await turingHelper.api.rpc.automationTime.generateTaskId(turingAddress, providedId);
+    const taskId = taskIdCodec.toString();
+
+    return { providedId, taskId, executionTime: nextExecutionTime };
+};
+
+const calculateTimeout = (executionTime) => (executionTime - moment().valueOf() / 1000 + LISTEN_EVENT_DELAY) * 1000;
+
+const main = async () => {
+    const turingHelper = new TuringHelper(TuringStaging);
+    await turingHelper.initialize();
+
+    const shibuyaHelper = new ShibuyaHelper(Rocstar);
+    await shibuyaHelper.initialize();
+
+    const turingChainName = turingHelper.config.key;
+    const parachainName = shibuyaHelper.config.key;
+    const parachainNativeToken = _.first(shibuyaHelper.config.assets);
+
+    console.log(`\nTuring chain key: ${turingChainName}`);
+    console.log(`Parachain name: ${parachainName}, native token: ${JSON.stringify(parachainNativeToken)}\n`);
+
+    const json = await readMnemonicFromFile();
+    const keyPair = keyring.addFromJson(json);
+    keyPair.unlock(process.env.PASS_PHRASE);
+
+    const account = new Account(keyPair);
+    await account.init([turingHelper, mangataHelper]);
+    account.print();
+
+    const parachainAddress = account.getChainByName(parachainName)?.address;
```

Comment:
Use `parachainAddress` instead of `sbyAddress` or `sdnAddress` so we can re-use the code.